src/org/labkey/test/util/DataRegionTable.java

```diff
@@ -137,6 +138,15 @@ public PagingWidget getPagingWidget()
         return _pagingWidget;
     }
 
+    public SetDefaultPageClass clicksetDefault()
+    {
+        if (!getCustomizeView().isPanelExpanded())
+        {
+            getViewsMenu().clickSubMenu(false, "Set Default");
```

Comment:
This menu option navigates.
```suggestion
            getViewsMenu().clickSubMenu(true, "Set Default");
```